docs/reference/search/search.asciidoc

```diff
@@ -87,6 +87,14 @@ both parameters are specified, only the query parameter is used.
 include::{docdir}/rest-api/common-parms.asciidoc[tag=from]
 +
 --
+[IMPORTANT]
```

Comment:
Some combo of only being able to page through 10k docs by default and the warning about deep paging would need to go replace these admonitions, as well.